PancakeSwap-Bots/Set-Masterchef-Settings/src/agent.ts

```diff
@@ -0,0 +1,31 @@
+import { Finding, HandleTransaction, TransactionEvent, getEthersProvider } from "forta-agent";
+import { providers } from "ethers";
+import NetworkData from "./network";
+import NetworkManager from "./network";
+import utils from "./utils";
+import abi from "./abi";
+
+const networkManager = new NetworkManager();
+
+export const initialize = (provider: providers.Provider) => async () => {
+  const { chainId } = await provider.getNetwork();
+  networkManager.setNetwork(chainId);
+};
+
+export const handleTransaction =
```

Comment:
please name this as `provideHandleTransaction` to align with our common practices